src/organisms/hooks/CameraAsyncHook.js

```diff
@@ -0,0 +1,30 @@
+import { useState, useEffect } from "react";
+
+export function cameraAsyncHook(requestedMedia) {
+  const [mediaStream, setMediaStream] = useState(null);
+
+  useEffect(() => {
+    async function enableVideoStream() {
+      try {
+        const stream = await navigator.mediaDevices.getUserMedia(
+          requestedMedia
+        );
+        setMediaStream(stream);
+      } catch (err) {
+        // Handle the error
```

Comment:
Adding an actual error will help us to debug easier should anything go wrong in the future.